Ports/AvailablePorts.md

```diff
@@ -1,280 +1,278 @@
-| [`wget`](wget/)                                     | GNU Wget                                                        | 1.21.3                   | https://www.gnu.org/software/wget/                                             |
-| [`which`](which/)                                   | GNU which                                                       | 2.21                     | https://www.gnu.org/software/which/                                            |
-| [`wireguard-tools`](wireguard-tools/)               | WireGuard Tools                                                 | 1.0.20210914             | https://www.wireguard.com/                                                     |
-| [`x264`](x264/)                                     | x264                                                            | baee400                  | https://www.videolan.org/developers/x264.html                                  |
-| [`x265`](x265/)                                     | x265                                                            | 3.5                      | https://bitbucket.org/multicoreware/x265_git/wiki/Home                         |
-| [`xash3d-fwgs`](xash3d-fwgs/)                       | Xash3D FWGS game engine                                         | 2022.12.26               | https://github.com/FWGS/xash3d-fwgs                                            |
-| [`xz`](xz/)                                         | xz                                                              | 5.2.5                    | https://tukaani.org/xz/                                                        |
-| [`yasm`](yasm/)                                     | Yasm Modular Assembler                                          | 1.3.0                    | https://yasm.tortall.net/                                                      |
-| [`zig`](zig/)                                       | Zig programming language                                        | 0.11.0-dev.670+f7fea080b | https://ziglang.org/                                                           |
-| [`zlib`](zlib/)                                     | zlib                                                            | 1.2.13                   | https://www.zlib.net/                                                          |
-| [`zsh`](zsh/)                                       | Z Shell (Zsh)                                                   | 5.8.1                    | https://www.zsh.org                                                            |
-| [`zstd`](zstd/)                                     | Zstandard                                                       | 1.5.2                    | https://facebook.github.io/zstd/                                               |
+| Port                                      | Name                                                          | Version                  | Website                                                              |
```

Comment:
Hm. The extra padding in this script was intentional so that adding a new Port with a longer name, longer description, or longer website name wouldn't cause a full table reformat commit.

Reply:
i feel that when generating itʼs not so painful; i could provide some minimal lengths that could be updated if needed but not really sure—at the moments itʼs just “snug-in”.

diff also does not look bad there (see the screenshot).
<img width="620" alt="example-diff" src="https://user-images.githubusercontent.com/25054764/216780028-d2bd488c-9345-4e66-b9ab-5c58f83b7ed5.png">

the length difference on table columns are: 10 chars for “port”, 2 chars for “name”, none for “version” and 10 chars for “website”. shall i provide this minimal length to match current formatting or is it okay to keep minimal lengths?

Reply:
It would be best to pad them out to the current lengths to minimize the change from this patch, and mitigate any full reformats from additional ports in the future. Ideally a "reformat using new script" commit in this PR would only be correcting incorrect information rather than completely redoing the table.

Reply:
iʼll accommodate for some reasonable, round length there. 👌
(shall i keep “version” intact or even avoid changes, or rather provide a new baseline?)

Reply:
Padding them out to a round number seems reasonable. I can only assume the previous lengths were "enough padding" to be a certain number larger than the (at the time) longest string per column.

Reply:
kept the previous ones for; have a look. 🙂